imgui.h

```diff
@@ -23,6 +23,13 @@
 #define IMGUI_API
 #endif
 
+// Calling conventions
+#if !defined(IMGUI_CALLBACK) && defined(_WIN32)
+#define IMGUI_CALLBACK __cdecl
+#else
+#define IMGUI_CALLBACK
+#endif
+
```

Comment:
Confused about how this blocks work, wouldn't statement in the else override whatever the user has specified?